content/docs/components/training/chainer.md

```diff
@@ -1,8 +1,8 @@
 +++
 title = "Chainer Training"
-description = "See v0.7 docs for instrustions on using Chainer for training"
+description = "See v0.6 docs for instrustions on using Chainer for training"
 weight = 4
 toc = true
 +++
 
-[Chainer](https://github.com/kubeflow/chainer-operator) is not supported in kubeflow 1.0
+[Chainer](https://github.com/kubeflow/chainer-operator) is not supported in kubeflow > 0.6
```

Comment:
Use a capital letter for **Kubeflow** as it's a product name. See the Kubeflow style guide: https://deploy-preview-1469--competent-brattain-de2d6d.netlify.com/docs/about/style-guide/#use-capital-letters-sparingly